conductr_cli/test/test_sandbox_features.py

```diff
@@ -1,7 +1,7 @@
 from unittest import TestCase
 from unittest.mock import call, patch, MagicMock
 from conductr_cli.sandbox_features import VisualizationFeature, LiteLoggingFeature,\
-    LoggingFeature, MonitoringFeature, \
+    LoggingFeature, MonitoringFeature, ProxyingFeature, \
```

Comment:
Good point - coverage was added for all of these cases except the first point. I moved the call to `sandbox_proxy.start` and there are tests on that method already to check the docker logic, so I think we're okay from a coverage standpoint.